sphinx/writers/html5.py

```diff
@@ -293,9 +292,13 @@ def add_secnumber(self, node):
                 if anchorname not in self.builder.secnumbers:
                     anchorname = ''  # try first heading which has no anchor
             if self.builder.secnumbers.get(anchorname):
-                numbers = self.builder.secnumbers[anchorname]
-                self.body.append('.'.join(map(str, numbers)) +
-                                 self.secnumber_suffix)
+                return self.builder.secnumbers[anchorname]
+        return None
+
+    def add_secnumber(self, node):
```

Comment:
Could you add type annotation here please?

Reply:
What do you mean?

Reply:
Pleasee append type annotation line like `# type: (nodes.Element) -> None` after function signature.